sites/engineering/features/content.feature

```diff
@@ -0,0 +1,277 @@
+  Then I should see the heading "Engineering Diversity Programs" in the "Content Head" region
+
+@safe
+Scenario: Verify that the Research and Faculty page has Faculty section
+  Given I am on "/people"
+  Then I should see the link "Faculty"
+
+@safe
+Scenario: Verify that the Faculty don't have titles
+  Given I am on "/people/faculty/grid"
+  Then I should see "" in the "div[class='field-content']" element
+
+@safe
+Scenario: Verify that the Emeritus Faculty have titles
+  Given I am on "/people/emeritus/grid"
+  Then I should see 1 or more "div[class='field-content']" elements
+
+@safe
+Scenario: Verify that the In Memoriam Faculty have titles
+  Given I am on "/people/in-memoriam/grid"
+  Then I should see 1 or more "div[class='field-content']" elements
+
+@safe
+Scenario: Verify that news stories have images
+  Given I am on "/news/recent-news"
+  Then the "div[class='postcard-left']" element should contain "img"
+  And the "div[class='postcard-image']" element should contain "img"
+
+@safe
+Scenario: Verify that there is a left nav on the news page
+  Given I am on "/news/recent-news"
+  Then I should see the "ul" element in the "Main Content" region
+
+@safe
+Scenario: Verify that there is a Twitter widget in the left sidebar
+  Given I am on "/news/recent-news"
+  Then I should see a "#block-bean-jumpstart-twitter-block" element in the "First sidebar" region
+
+#Test should be written for news article pages
+#Scenario: Verify there is no text that says "Banner" nor "Banner Overlay", has "See more news" and no left nav
+#  Given I am on "/news/../"
+#  And I should not see "Banner" in the "#block-ds-extras-banner" element in the "Top Full Width" region
+#  And I should not see "Banner Overlay" in the "#block-ds-extras-banner" element in the "Top Full Width" region
+#  And I should see the text "See more news"
+#  And I should not see a "#block-bean-jumpstart-twitter-block" element
+
+@safe
+Scenario: Verify there is no text that says "Banner" nor "Banner Overlay", has "See more news" and no left nav
+  Given I am on "/news/meet-jackrabbot-social-robot"
+  Given I am on "/news/stanford-innovation-self-healing-skin-easier-drink-pouring-and-new-kind-jump-rope"
+  Given I am on "/news/donald-allen-dunn-1925-2011"
+  Given I am on "/news/sarah-moore-doctoral-student-bioengineering"
+  Given I am on "/news/arun-majumdar-what-biggest-challenge-21st-century"
+  Given I am on "/news/stanford-researchers-survey-protein-family-helps-brain-form-synapses"
+  Given I am on "/news/stanford-engineer-produces-free-braille-writer-app/"
+  And I should not see "Banner" in the "#block-ds-extras-banner" element in the "Top Full Width" region
+  And I should not see "Banner Overlay" in the "#block-ds-extras-banner" element in the "Top Full Width" region
+  And I should see the text "See more news"
+  And I should not see a "#block-bean-jumpstart-twitter-block" element
+  And I should not see an "div[id='sidebar-first']" element
+
```

Comment:
If you want to have it test the banner and banner overlay for each of these pages, you'll need to have test for each page, not just one at the end.